### PR TITLE
Added a new field in load from seed

### PR DESCRIPTION
**Pull request by nakulpandey**

### CODE REVIEW

File: src/gui/static/src/app/app.loadWallet.ts [hunk 1]
```diff
@@ -705,19 +710,46 @@ export class LoadWalletComponent implements OnInit {
         .map((res:Response) => res.json())
         .subscribe(
             response => {
-                //Hide load wallet seed popup
-                this.loadSeedIsVisible = false;
-                //Load wallet for refresh list
-                this.loadWallet();
-            },
-            err => {
-                if(err._body.indexOf("duplicate wallet ") !=-1){
```
Comment (iketheadore) [inline src/gui/static/src/app/app.loadWallet.ts]: The duplicate wallet checking is gone, will stuck when try to load wallet from same seed.

File: src/gui/static/src/app/templates/wallet.html [hunk 2]
```diff
@@ -596,20 +596,26 @@ <h4 class="modal-title">Create New Wallet</h4>
                 <button type="button" (click)="hideLoadSeedWalletPopup()" class="close">&times;</button>
                 <h4 class="modal-title">Load wallet by seed</h4>
             </div>
-            <form class="form-horizontal" (ngSubmit)="createWalletSeed(walletname.value, walletseed.value)">
+            <form class="form-horizontal" (ngSubmit)="createWalletSeed(walletname.value, walletseed.value, addressesToCreate.value)">
                 <div class="modal-body">
                     <div class="form-group">
-                        <label class="col-sm-2 control-label">Name:</label>
+                        <label class="col-sm-3 control-label">Name:</label>
```
Comment (iketheadore) [inline src/gui/static/src/app/templates/wallet.html]: `New Wallet` use `Label` field, we should have consistent field name, I propose change the `Label` field in `New Wallet` to `Name`.

**Comment by iketheadore** [comment]
`Create New Wallet` from the same seed will show this:

image

**Comment by nakulpandey** [comment]
@iketheadore please check

**Pull request merged**